browser/ui/browser_commands.cc

```diff
@@ -30,6 +30,7 @@
 #include "brave/browser/speedreader/speedreader_service_factory.h"
 #include "brave/browser/speedreader/speedreader_tab_helper.h"
 #include "brave/components/speedreader/speedreader_service.h"
+using DistillState = speedreader::SpeedreaderTabHelper::DistillState;
```

Comment:
pls move this down to the function

Reply:
done

Reply:
not sure

Reply:
i did it in the wrong place. fixed with new changes